src/pyOpenMS/pyopenms/_dataframes.py

```diff
@@ -339,6 +351,7 @@ def get_massql_df(self):
         'precmz': mass to charge of the precursor ion
         'ms1scan': number of the corresponding MS1 spectrum
         'charge': charge of the precursor ion
+        'ion': the ionic mobility of a peak if ion parameter is True
 
```

Comment:
Please add a Parameter section to describe the new ion parameter including type bool.